### ISSUE TITLE
Suggestion : Running phpcbf and phpcs

### ISSUE THREAD

**Issue by harikt** [squizlabs/PHP_CodeSniffer#623]
Hi, 

I have a suggestion when we run 

```php
php phpcbf.phar 
```

or 

```php
php phpcs.phar 
```

it just hangs with no output. My suggestion is to show the help rather it shows like it is not working.
So users don't need to use ctrl + c to terminate .

**Comment by gsherwood** [comment]
They hang like that because they are waiting for STDIN, which some plugins and wrappers provide to the tool.

In the 3.x version, I have changed this behaviour so that they print the help by default and require plugins to use a command line switch to tell PHPCS/CBF that they are passing in STDIN. But this is a big BC break, so I'm not back-porting it to the 2.x version.

This is the commit that introduced that change, although it has a few other things in it as well: https://github.com/squizlabs/PHP_CodeSniffer/commit/5d0784a3351d7e04188005c8eec9dd829d5d481e

So the summary: I agree with you and I've made that change, but I have to wait until 3.0 so I can release it as a BC break.
